src/NuGet.Clients/NuGet.VisualStudio/NuGet.VisualStudio.csproj

```diff
@@ -18,6 +18,12 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
     <DocumentationFile>$(OutputPath)\NuGet.VisualStudio.XML</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <DocumentationFile>$(OutputPath)\NuGet.VisualStudio.XML</DocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Mono Release'">
+    <DocumentationFile>$(OutputPath)\NuGet.VisualStudio.XML</DocumentationFile>
```

Comment:
No clue :) seems like the default when checking the checkbox, and the already existing nuspec had it capitalized. Will change to lowercase everywhere.